CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+### Changed
+
+- Add support for the versioned OS downloads, see
+`resin.models.os.{getSupportedVersions, getMaxSatisfyingVersion, formatVersion}`.
```

Comment:
`formatVersion` should go from here, now that it's out of the code.

Docs need updating too.